client.go

```diff
@@ -99,6 +100,13 @@ func SetNonce(nonce string) AuthCodeOption {
 	}
 }
 
+// AddScope adds an additional scope request to this URL only
+func AddScope(scope string) AuthCodeOption {
```

Comment:
Could be `AddScopes(...string)` or `[]string` if you want parity to the client-level one.

I don't feel strongly one way or the other though.

Reply:
Switched it over